Makefile

```diff
@@ -0,0 +1,13 @@
+all: app editor web
+
+app:
+	docker build -f ./app/Dockerfile.base -t driver/app:latest ./app
+	docker build -f ./app/Dockerfile.development -t driver/app-dev:latest ./app
+
+editor:
+	docker build -f ./schema_editor/Dockerfile -t driver/editor:latest ./schema_editor
+
+web:
+	docker build -f ./web/Dockerfile -t driver/web:latest ./web
+
+.PHONY: all app editor web
```

Comment:
Added a section to the `README` in https://github.com/WorldBank-Transport/DRIVER/commit/23588d708006cac7bae3de46ff1088b321df8fa3.